swagger.yaml

```diff
@@ -1,42 +1,236 @@
+    name: edition
+    description: "An edition of a dataset"
+    in: path
+    required: true
+    type: string
+  version:
+    name: version
+    description: "A version of a dataset"
+    in: path
+    required: true
+    type: string
+  dimension_options:
+    description: "The name of the dimension option and a single value; each option (dimension) and corresponding value (code) must exist against the version - e.g. `age=30` or one of the dimension options can be represented by a wildcard value `*` e.g. `geography=*`"
+    name: "<dimension_options>"
+    in: query
+    required: true
+    type: string
+
+securityDefinitions:
+  FlorenceAPIKey:
+    name: florence-token
+    description: "API key used to allow florence users to create and query the progress of importing a dataset"
+    in: header
+    type: apiKey
+  InternalAPIKey:
+    name: internal-token
+    description: "API key used to allow only internal services to update the state of an import job"
+    in: header
+    type: apiKey
+  DownloadServiceAPIKey:
+    name: x-download-service-token
+    description: "API key used to allow the download service to access public and private links to a download"
+    in: header
+    type: apiKey
 paths:
-  /hello:
+  /datasets/{id}/editions/{edition}/versions/{version}/observations:
     get:
       tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
-      produces:
-        - application/json
+      - "Public"
+      summary: "Get specific observations"
+      description: "Get observations from a version of the dataset. By providing
+      a single option for each dimension, a single observation will be returned.
+      A wildcard (*) can be provided for one dimension, to retrieve a list of
+      observations."
+      parameters:
+        - $ref: '#/parameters/edition'
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/version'
+        - $ref: '#/parameters/dimension_options'
       responses:
         200:
-          description: OK
+          description: "Json object containing all metadata for a version"
           schema:
-            $ref: "#/definitions/HelloResponse"
+            $ref: '#/definitions/ObservationsEndpoint'
+        400:
+          description: |
+            Invalid request, reasons can be one of the following:
+              * query parameters missing expected dimensions
+              * query parameters contain incorrect dimensions
+              * too many query parameters are set to wildcard (*) value; only one query parameter can be equal to *
+        404:
+          description: |
+            Resource not found, reasons can be one of the following:
+              * dataset id was incorrect
+              * edition was incorrect
+              * version was incorrect
+              * observations not found for selected query paramaters
         500:
           $ref: '#/responses/InternalError'
 
 responses:
+  ConflictError:
```

Comment:
These responses are not referenced in this swagger spec with the exception of InternalError, can remove all others